device/celestica/x86_64-cel_silverstone-r0/sonic_platform/psu.py

```diff
@@ -33,20 +33,15 @@
 PSU_LED_GREEN_CMD = "0x01"
 PSU_LED_AMBER_CMD = "0x02"
 
-PSU1_VOUT_SS_ID = "0x36"
-PSU1_COUT_SS_ID = "0x37"
-PSU1_POUT_SS_ID = "0x38"
-PSU1_STATUS_REG = "0x39"
-
-PSU2_VOUT_SS_ID = "0x40"
-PSU2_COUT_SS_ID = "0x41"
-PSU2_POUT_SS_ID = "0x42"
-PSU2_STATUS_REG = "0x2f"
-
 PSU1_FRU_ID = 3
 
 SS_READ_OFFSET = 0
 
+PSU_VOUT_SS_ID = {"PSU1_VOUT_SS_ID": "0x36", "PSU2_VOUT_SS_ID": "0x40"}
```

Comment:
>PSU_VOUT_SS_ID [](http://example.com/codeflow?start=0&length=14)

Is an array better than a dict?

Reply:
I believe a dict is better than array since it is easier to get value of the key and faster to lookup.

Reply:
Fixed.